enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,305 @@
+[Direct Link][direct-link-website] offering. This will allow for both the ingress load balancer and the cluster nodes to
+talk over the private network on a high-speed link.
+7. Bootstrapping completes as normal.
+8. The workers will be added using the openshift/[cluster-\*machine-\*] controllers and operators based on the openshift
+MAPI model. A Power VS controller needs to be implemented and hooked into the [OpenShift Machine
+Operator][openshift-machine-operator].
+9. The cluster teardown can make use of the existing [go client for Power][power-go-client] via `openshift-install
+destroy cluster`.
+
+### User Stories
+
+- A user of this installer would like to see a cluster running within an hour of kicking off the installer.
+- The expectation is that this is fully automated.
+
+### Implementation Details/Notes/Constraints
+
+#### Existing APIs in the Upstream
+There is a working [implementation of a machine API](https://github.com/openshift-powervs/cluster-api-provider-powervs)
+we've created for testing purposes. Though not ready for upstream, many have been tested and found to be working, such
+as mapi and nodeupdate controller. This was tested with machineset as well, so things look promising.
+
+#### CoreOS OVA Considerations
+There were two main implementation ideas for making the CoreOS OVA boot image available to the Virtual Server instance.
+
+1. Import the Image from a Cloud Store This is the implementation we believe is best, since it works most similarly to
+other cloud environments. In this setup, the CoreOS build pipeline would publish an OVA image to an IBM Cloud Object
+Storage (COS) bucket. Then, we will use the regular IBM Cloud APIs to import the image to the instance. The drawback to
+this approach is the the current implementation is very slow. In its current state, it takes about 30 minutes to import
+an 120 Gb boot image. This is too long to meet our target of being able to install a whole cloud within the hour. One of
+the reasons this is slow is because we cannot do the image resizing after the copy, since it's writing directly to a
+sized disk. Once imported, creating the nodes is fast, since the other disks are flashed from the snapshot. For now, we
+asked the team to look into meeting the requirement that importing the image can take 5 minutes or less. Another bit
+we're working on is defining the implementation details for how the image will be imported from the COS bucket to the
+PowerVS service. Currently, the terraform API doesn't support importing an image from bucket. We are looking into if it
+is possible to update the terraform provider to do this with support for optional keys to support both private and
+public buckets. IBM Cloud Object Storage supports public buckets, but the terraform provider API will need an update to
+make keys optional in order to support this workflow.
+
+At this point, we are still determining whether this will happen as part of the installer or the terraform provider.
```

Comment:
this should be available via the terraform provider. and we can strikethrough (or delete) 2) below